src/Common/tests/XunitTraitsDiscoverers/Discoverers/ActiveIssueDiscoverer.cs

```diff
@@ -23,7 +23,7 @@ public class ActiveIssueDiscoverer : ITraitDiscoverer
         public IEnumerable<KeyValuePair<string, string>> GetTraits(IAttributeInfo traitAttribute)
         {
             IEnumerable<object> ctorArgs = traitAttribute.GetConstructorArguments();
-            Contract.Assert(ctorArgs.Count() >= 2);
+            Debug.Assert(ctorArgs.Count() >= 2);
```

Comment:
Actually if you don't attach debugger it just fails fast and doesn't display anything anywhere (not even standard error). In theory this will never happen, although if we ever add another constructor to ActiveIssue it might happen during development I believe